README.md

```diff
@@ -18,21 +18,52 @@
 
 ## :rocket: Getting started
 
-### Install Iter8 CLI
+### Using Iter8 Action
```

Comment:
Could you please have two sections under Getting started?

### Local  usage (where we want to retain the previous documentation like Install Iter8, HTTP and gRPC load tests), and 

### GitHub Actions usage (where we want to add your new documentation).

Eventually, we will also include Kuberenetes usage above.

Reply:
I guess I think I do have these sections albeit with different titles and in the reverse order. The order should reflect the common use for people who read the README. Of course, since I wrote actions, I think it the more common use case :) Practically speaking, github action users will come via the readme since that is what the marketplace will give them. Will CLI users start with readme or doc?

Reply:
The CLI usage is IMO, the more predominant usage. We can always reassess thee order. If GH actions does turn out to be more predominant, we can switch. But we should start with the simplest, which is CLI.

Reply:
The README is not really meant exclusively for GitHub Actions users. The README will also determine... star worthiness or fork worthiness or use worthiness of the project as a whole.

Since we decided to club the GH Action as part of this repo, it is not right to look at the README in isolation only w.r.t. GH Action.

***

Anyway, please use the section titles ## Local Usage and ## GitHub Actions Usage, which would make it convenient for me to add ## Kuberntes Usage (the last is coming soon).

Reply:
Then local first.